src/main/java/org/terasology/launcher/game/GameManager.java

```diff
@@ -87,30 +83,18 @@ public void install(GameRelease release, ProgressListener listener) throws IOExc
         }
     }
 
+    /**
+     * @deprecated Use {@link DownloadUtils#download(RemoteResource, Path, ProgressListener)} instead.
+     */
+    @Deprecated
```

Comment:
am lacking experience here, but the method is private. why deprecating it?

Reply:
Not sure whether it actually has any effect other than your IDE telling you this is deprecated 🤷  I kinda see it as a TODO to replace the call side of this method with what is written in the deprecation note. 
That way, I don't change any interfaces now, and can make this change in a follow up PR to keep the PRs smaller.

Reply:
can we add a deprecation note to the upper one, too?

Reply:
There was only a single use of that other method, so I just inlined it and removed the method in [482ba65](https://github.com/MovingBlocks/TerasologyLauncher/pull/712/commits/482ba65fe0bfe8597d2b1da2f6e1ddc92e44184d).